### PR TITLE
add docs to basket tag, recipe details

### PR DESCRIPTION
**Pull request by diarmuidmcg**

### CODE REVIEW

**Comment by diarmuidmcg** [comment]
Current dependencies on/for this PR:
* main
  * **PR #11** <a href="https://app.graphite.dev/github/pr/miamtech/MiamNeutraliOSFramework/11" target="_blank">Graphite</a> 
    * **PR #12** <a href="https://app.graphite.dev/github/pr/miamtech/MiamNeutraliOSFramework/12" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/miamtech/MiamNeutraliOSFramework/12?utm_source=stack-comment).

**Comment by diarmuidmcg** [comment]
### Merge activity

* **Oct 25, 10:59 AM**: @diarmuidmcg started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/miamtech/MiamNeutraliOSFramework/12).

File: Sources/MiamNeutraliOSFramework/DefaultPageParameters/MealPlanner/Results/MealPlannerResultsViewOptions.swift [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        emptyResults: TypeSafeMealPlannerResultsEmpty = TypeSafeMealPlannerResultsEmpty(MiamNeutralMealPlannerResultsEmpty()),
+        empty: TypeSafeEmpty = TypeSafeEmpty(DefaultEmptyView()),
+        loading: TypeSafeLoading = TypeSafeLoading(DefaultLoadingView()),
+        background: TypeSafeBackground = TypeSafeBackground(DefaultBackgroundView())
+    ) {
+        self.toolbar = toolbar
+        self.footer = footer
+        self.recipeCard = recipeCard
+        self.recipeCardLoading = recipeCardLoading
+        self.placeholder = placeholder
```
Comment (ThibaultMiam) [inline Sources/MiamNeutraliOSFramework/DefaultPageParameters/MealPlanner/Results/MealPlannerResultsViewOptions.swift:51]: recipeCardPaceholder ?

Comment (diarmuidmcg) [reply Sources/MiamNeutraliOSFramework/DefaultPageParameters/MealPlanner/Results/MealPlannerResultsViewOptions.swift:51]: you think recipeCardPlaceholder is a better than just "placeholder"?

**Pull request merged**